stable/metacontroller/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,9 @@
+{{ if .Values.rbac.create }}
```

Comment:
https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md
this only depends on serviceAccount.create